stream.go

```diff
@@ -68,6 +68,7 @@ func (s *Stream) ReadSome() (instruction []byte, err error) {
 		return
 	}
 
+	buffer := make([]byte, MaxGuacMessage)
```

Comment:
So all of this looks reasonable saying that I'm not exactly sure why buffer in this instance or instructions stay []byte arrays when we are using runes for UTF8 data but I'm assuming there is a reason for that

Reply:
`buffer` here needs to be `[]byte` since it's reading from `net.Conn` which reads bytes from the wire.

`instruction` should definitely become something else, probably `[]rune`, but we'll need to cut a major release to make those changes since it's a part of the function signature. There are also at least one interface that will need to change too. 

I figure we make it correct internally and then fix the function signatures in the next release.